float32 set_point
---
string result